Makefile

```diff
@@ -12,128 +12,97 @@ test: test-unit ## run tests
 
 .PHONY: test-unit
 test-unit: ## run unit tests, to change the output format use: GOTESTSUM_FORMAT=(dots|short|standard-quiet|short-verbose|standard-verbose) make test-unit
-	@./scripts/test/test-unit
+	@./scripts/test/unit.sh
 
 .PHONY: test-coverage
 test-coverage: ## run test coverage
-	@./scripts/test/test-coverage
+	@./scripts/test/coverage.sh
 
 .PHONY: test-coverage-html
 test-coverage-html: ## run test coverage and generate cover.html
-	@./scripts/test/test-coverage-html
+	@./scripts/test/coverage-html.sh
 
 # ============= LINT =============
 .PHONY: lint
 lint: ## run linter
-	@./scripts/lint
+	@./scripts/test/lint.sh
 
 # ============= MOCKS =============
 .PHONY: mock
 mock: mock-monitorable mock-pkg
 
 .PHONY: mock-monitorable
 mock-monitorable: ## generate mocks of monitorable sub-directories
-	@./scripts/mock/mock-monitorable
+	@./scripts/mock/monitorable.sh
 
 .PHONY: mock-pkg
 mock-pkg: ## generate mocks of pkg directory listed in scripts/mock/mock-pkg
-	@./scripts/mock/mock-pkg
+	@./scripts/mock/pkg.sh
 
 # ============= BUILDS =============
 .PHONY: build
-build: ## build executable for current environment
-	@./scripts/build/rice
-	@./scripts/build/build
+build: package-front ## build executable for current environment
+	@./scripts/build/build.sh
 
 .PHONY: build-cross
-build-cross: ## build all executables
-	@./scripts/build/rice
-	@./scripts/build/build linux
-	@./scripts/build/build windows
-	@./scripts/build/build macos
-	@./scripts/build/build raspberrypi
-
-.PHONY: build-linux
-build-linux: ## build executable for Linux
-	@./scripts/build/rice
-	@./scripts/build/build linux
+build-cross: package-front ## build all executables
+	@./scripts/build/build.sh linux/amd64
+	@./scripts/build/build.sh linux/ARMv5
```

Comment:
just `arm` seems OK
maybe one day we will build `arm64`, but I think that's it.